BootloaderCorePkg/Stage2/Stage2Hob.c

```diff
@@ -604,6 +604,32 @@ BuildUniversalPayloadHob (
   BuildUpldPciRootBridgeHob ();
 }
 
+/**
+  Update graphics hobs.
+
+**/
+VOID
+EFIAPI

```

Comment:
This UpdateGraphicsHob () function is only used inside Stage2.inf, it is not necessary to use EFIAPI which is mainly used for the interface with other modules.

Reply:
OK, let me remove it.